src/targets/browser/index.jsx

```diff
@@ -13,7 +13,7 @@ import {
 import { CozyProvider } from 'cozy-client'
 import { I18n } from 'cozy-ui/react/I18n'
```

Comment:
```suggestion
import { I18n } from 'cozy-ui/transpiled/react/I18n'
```

Je te conseille de faire un chercher / remplacer, sinon tu vas te retrouver avec des soucis de hooks tu vas pas comprendre pourquoi

Reply:
En effet, j'ai eu le problème juste avant avec cet import `cozy-ui/transpiled/react/hooks/useBreakpoints`..